File: 2D-World-Simulator-CPP/OPP-CPP/Wolf.cpp
#include "Wolf.h"

Wolf::Wolf(World* world, Point point) : Animal(world, point) {

	 this->setOrganismSign('W');
	 this->setStrength(9);
	 this->setInitiative(5);
	 this->setOrganismSpecie(OrganismSpecie::WOLF);

}

File: 2D-World-Simulator-Java/OPP-Java/src/op/JavaProject/Organisms/Animals/Wolf.java
package op.JavaProject.Organisms.Animals;

import op.JavaProject.Functionality.BoardPoint;
import op.JavaProject.Functionality.Species;
import op.JavaProject.Game.Animal;
import op.JavaProject.Game.World;

import java.awt.*;


public class Wolf extends Animal {

    public Wolf(World organismWorld, BoardPoint point) {
        super(organismWorld, point);
        this.strength = 9;
        this.initiative = 5;
        this.organismSign = "W";
        this.organismSpecie = Species.WOLF;
        this.organismColor = new Color(147, 147, 147);
    }
}
